eng/common/scripts/modules/git-api-calls.psm1

```diff
@@ -0,0 +1,81 @@
+# Methods for interacting with the github or azure repo APIs
+# Retrieves the list of all tags that exist on the target repository
+# Changes 1: Pass full API URL instead of formulating it within the method
+# Changes 2: Extend method for git Repos
+function GetExistingTags($apiUrl, $repoType = "github", $headers = $null) {
```

Comment:
In consistent bracing. I'd suggest always putting the opening and closing braces on their own line, especially for larger blocks like functions and try/catch. For one line if statements having the opening brace on the same line as the if it is more acceptable because you can see the full context.